scripts/analyze.sh

```diff
@@ -103,26 +94,20 @@ if [ $? -eq 0 ]; then
     >&2 echo Using module: PWM
     MODULES+=" -DBUILD_MODULE_PWM"
     echo "CONFIG_PWM=y" >> prj.conf.tmp
-    echo "CONFIG_PWM_QMSI_NUM_PORTS=4" >> prj.conf.tmp
```

Comment:
Don't remove this line. It still works, and this is still required to make the A101 PWM pins work (apart from the first one).